testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestCreateDropSchema.java

```diff
@@ -0,0 +1,145 @@
+    {
+        useIceberg();
+        String schemaName = schemaName("drop_schema_empty_default_location");
+        String schemaDir = format("%s/%s.db/", warehouseDirectory, schemaName);
+
+        createSchema(schemaName);
+        assertFileExists(schemaDir, true, "schema directory exists after create schema");
+        assertQuerySucceeds("DROP SCHEMA " + schemaName);
+        assertFileExists(schemaDir, false, "schema directory exists after drop schema");
+    }
+
+    @Test(groups = ICEBERG)
+    public void testDropSchemaWithExternalFileAndLocation()
+    {
+        useIceberg();
+        String schemaName = schemaName("drop_schema_nonempty_location");
+        String schemaDir = warehouseDirectory + "/schema-with-nonempty-location/";
+
+        // Create file in schema directory before creating schema
+        String externalFile = schemaDir + "external-file";
+        hdfsClient.createDirectory(schemaDir);
+        hdfsClient.saveFile(externalFile, "");
+
+        createSchema(schemaName, schemaDir);
+        assertFileExists(schemaDir, true, "schema directory exists after create schema");
+        assertQuerySucceeds("DROP SCHEMA " + schemaName);
+        assertFileExists(schemaDir, true, "schema directory exists after drop schema");
+
+        assertFileExists(externalFile, true, "external file exists after drop schema");
+        hdfsClient.delete(externalFile);
+        hdfsClient.delete(schemaDir);
+    }
+
+    @Test(groups = ICEBERG)
+    public void testDropSchemaWithExternalFileWithoutLocation()
+    {
+        useIceberg();
+        String schemaName = schemaName("drop_schema_nonempty_default_location");
+        String schemaDir = format("%s/%s.db/", warehouseDirectory, schemaName);
+
+        // Create file in schema directory before creating schema
+        String externalFile = schemaDir + "external-file";
+        hdfsClient.createDirectory(schemaDir);
+        hdfsClient.saveFile(externalFile, "");
+
+        createSchema(schemaName);
+        assertFileExists(schemaDir, true, "schema directory exists after create schema");
+        assertQuerySucceeds("DROP SCHEMA " + schemaName);
+        assertFileExists(schemaDir, true, "schema directory exists after drop schema");
+
+        assertFileExists(externalFile, true, "external file exists after drop schema");
+        hdfsClient.delete(externalFile);
+        hdfsClient.delete(schemaDir);
+    }
+
+    private void assertFileExists(String path, boolean exists, String description)
+    {
+        assertThat(hdfsClient.exist(path)).as(description).isEqualTo(exists);
+    }
+
+    private static void assertQuerySucceeds(String query)
+    {
+        try {
+            onTrino().executeQuery(query);
+        }
+        catch (QueryExecutionException e) {
+            fail(format("Expected query to succeed: %s", query), e.getCause());
+        }
+    }
+
+    private void useIceberg()
+    {
+        onTrino().executeQuery("USE iceberg.default");
```

Comment:
I'd use fully qualified names instead, but whatever

Reply:
I had that at first, but then if I (or any future developer) forgot to qualify the names anywhere, the tests might pass without actually using Iceberg.

It also makes the variable declarations in the tests a little less nice, because you need the unqualified schema name for the default schema location.

Reply:
> 

we should remove default catalog/schema from tempto configuration here
https://github.com/trinodb/trino/blob/4f67c9055626cbd893f97d7cc4ac5d36be399c3f/testing/trino-product-tests/src/main/resources/tempto-configuration.yaml#L34

@jirassimok can you work on that, separately?